src/api/ucc_status.h

```diff
@@ -31,6 +31,7 @@ typedef enum {
     UCC_ERR_INVALID_PARAM               =   -3,
     UCC_ERR_NO_MEMORY                   =   -4,
     UCC_ERR_NO_RESOURCE                 =   -5,
+    UCC_ERR_NO_MESSAGE                  =   -6,
```

Comment:
not clear what this means... could you please add a comment?

Reply:
This is general purpose return code without explicit error specification. Same exists in ucx. I'll add comment. (this is will be another PR)